edgeless_dataplane/src/handle.rs

```diff
@@ -177,7 +177,7 @@ impl DataplaneProvider {
 
         log::info!("coap port {}", port);
 
-        let _coap_server = tokio::spawn(edgeless_api::coap_impl::CoapInvocationServer::run(
+        let _coap_server = tokio::spawn(edgeless_api::coap_impl::invocation::CoapInvocationServer::run(
             clone_provider.lock().await.incomming_api().await,
```

Comment:
Sooner or later we should fix this typo (s/incomming/incoming). I think it might be safe to do so already by pushing to this branch just before rebasing the corresponding PR.